app/src/test/java/com/github/h3lp3rs/h3lp/database/EmergencyInfoRepositoryTest.kt

```diff
@@ -16,6 +16,7 @@ class EmergencyInfoRepositoryTest {
         EmergencyInfoRepository(MockDatabase())
```

Comment:
This (underlying) class still gives a coverage of F on codeclimate. Interesting...